constitution.md

```diff
@@ -27,7 +27,7 @@
 
 1. Eligibility
 
-    Members shall be selected by methods adopted by the Executive Committee.
+    To be a member of OS2G, one must be a student at the University of Nebraska—Lincoln and attend at least four meetings in a single semester. A member is eligible to vote for officers in a given semester if they have attended at least four meetings in that semester, prior to the annual meeting.
```

Comment:
In all three locations where it says "annual meeting", it may be good to specify the "annual election meeting"